docs/balena-cli.md

```diff
@@ -367,6 +367,14 @@ fleet name, slug (preferred), or numeric ID (deprecated)
 
 open fleet dashboard page
 
+#### --fields FIELDS
+
+only show provided fields (comma-separated)
```

Comment:
Let's change the commit to `fleet: Add --fields and --json options`, to be consistent w/ how the changelog looks.